include/libdnf-cli/output/key_value_table.hpp

```diff
@@ -36,7 +36,6 @@ class KeyValueTable {
     ~KeyValueTable();
     void print();
 
-protected:
```

Comment:
Do we really need it to make it public?

Reply:
Yes. It is the actual interface for using the class. When it was protected, the only way to use it was to inherit from the class.

I had no need for inheriting, I just wanted to use the class as it is (see the history info code).

Reply:
Well advisory  can access it internally. It does not need to be as a part of API.

Reply:
Is it possible to create formatting functions like `print_transaction_table(Transaction & transaction)` that will use those functions internally?

Reply:
It makes no sense to try to hide this API, it is _the_ API that should be public for a libdnf-cli output class.